Tasks/DockerV1/task.json

```diff
@@ -12,17 +12,29 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
-        "Patch": 4
+        "Minor": 1,
+        "Patch": 0
     },
     "demands": [],
     "preview": "true",
     "releaseNotes": "Simplified the task by:<br/>&nbsp;- Providing an option to simply select or type a command.<br/>&nbsp;- Retaining the useful input fields and providing an option to pass the rest as an argument to the command.",
     "groups": [
+        {
+            "name": "containerRegistry",
+            "displayName": "Container Registry",
+            "isExpanded": true,
```

Comment:
Should this remain collapsed?

Reply:
To keep consistance with other task i kept expand true.